src/scyjava/__init__.py

```diff
@@ -1022,6 +1028,12 @@ def _stock_py_converters() -> typing.List:
             converter=JavaIterator,
             priority=Priority.NORMAL - 1,
         ),
+        # Path converter
+        Converter(
+            predicate=lambda obj: isinstance(obj, JavaPath),
+            converter=lambda obj: Path(str(obj.toString())),
```

Comment:
Why did you choose `java.nio.file.Path` rather than `java.io.File`? Any specific reason, other than the naming and feature set being a closer match to `pathlib.Path`? I know that `java.io.File` is not as nice, but it is more established, and it's what we have been using in scijava-common everywhere else till now. With `java.nio` I always worry about whether it will be available in whatever weird JVM people are trying to use. That said, I just checked for Android, and it seems both [File](https://developer.android.com/reference/java/io/File) and [Path](https://developer.android.com/reference/java/nio/file/Path) are available on Android, so... 🤷 It's probably fine.

Reply:
> Any specific reason, other than the naming and feature set being a closer match to `pathlib.Path`?

Yes, this was the reason.

> With `java.nio` I always worry about whether it will be available in whatever weird JVM people are trying to use.

Oh, is that an issue?

Reply:
> Oh, is that an issue?

Might have been 20 years ago when `java.nio` was new, back in Java 1.4. I remember we had to add case logic to Bio-Formats in the mid-to-late 2000s in case the memory-mapped I/O subsystem was not available to that particular JVM. 🧓

The `java.nio.file` sub-package was added in Java 7, so only 11 years old, so maybe some JVM somewhere still doesn't support it, who knows. The [Google GWT](https://en.wikipedia.org/wiki/Google_Web_Toolkit) project for example was stuck on Java 6 for quite some time after Java 8 features were available, so wouldn't have had `Path`, but apparently GWT gained Java 7 support [in 2014](https://stackoverflow.com/a/15693528), and some limited Java 8 support in late 2016, and by now is probably even further along.

But hopefully that gives you an idea why I am concerned about these things.